.cirrus/azure_app_verify_and_publish.sh

```diff
@@ -14,8 +14,6 @@ PSQL_VERSION="${PSQL_VERSION:-11.14.0-debian-10-r22}" # PostgreSQL version used
 # Azure Container Registry (ACR) details
 # This should match the 'registryServer' in your manifest.yaml
 ACR_REGISTRY="${AZURE_ACR_REGISTRY:-}"  
-ACR_USERNAME="${AZURE_ACR_USERNAME:-}" # Use environment variable or provide directly
```

Comment:
The `build_azure_staging` is still failing due to an additional docker login we have to perform prior to running the cpa from the container. We should provide the username and password back here..